documentation/specs/TabPanels.md

```diff
@@ -0,0 +1,163 @@
+# `TabPanels` Component Specification
```

Comment:
the name, along with the previous `TabNav`, follows the Github pattern of `TabNav` and `TabPanels`.

i don't love that the consumer has to think about what version of the component to use but the implementations vary so wildly that i don't see a realistic path toward making them one (and likely why most component libraries have them separately [except for MUI (kind of)]. maybe it's good to give thought to the right pattern to use per instance, though, given that navigational behavior is critical in an app.

Reply:
I agree, don't love it for needing to think about which to use but maybe we could add a note to each on usage and link to the other